dcos_launch/gce.py

```diff
@@ -0,0 +1,87 @@
+import os
+import logging
+import dcos_launch.util
+from dcos_test_utils import gce
+from retrying import retry
+
+log = logging.getLogger(__name__)
+
+
+class BareClusterLauncher(dcos_launch.util.AbstractLauncher):
+    # Launches a homogeneous cluster of plain GMIs intended for onprem DC/OS
+    def __init__(self, config):
+        # config = get_hardcoded_basic_config(os_name, instance_count)
+        self.cwd = os.path.dirname(os.path.realpath(__file__))
+        gce_folder = os.path.expanduser('~') + '/.gce'
+        credentials_path = gce_folder + '/credentials.json'
+        self.keys_path = gce_folder + '/keys'
+        self.gce_wrapper = gce.GceWrapper(credentials_path)
+        self.private_key_path = self.keys_path + '/' + config['deployment_name'] + '.priv'
```

Comment:
I would prefer to see the `.format` style here: `'{}/{}.priv'.format(self.keys_path, config['deployment_name'])`